### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by Daniil-Pankieiev**

### CODE REVIEW

File: cinema/views.py [hunk 1]
```diff
@@ -21,7 +23,7 @@
     MovieSessionDetailSerializer,
     MovieListSerializer,
     OrderSerializer,
-    OrderListSerializer,
+    OrderListSerializer, MovieImageSerializer,
```
Comment (prochigor) [inline cinema/views.py]: refactor this

File: user/serializers.py [hunk 2]
```diff
@@ -22,3 +24,38 @@ def update(self, instance, validated_data):
...
+        password = attrs.get("password")
+
+        if email and password:
+            user = authenticate(
+                request=self.context.get("request"),
+                email=email,
+                password=password
+            )
+
+            # The authenticate call simply returns None for is_active=False
```
Comment (prochigor) [inline user/serializers.py]: if it like doc string, it writes with """doc""", not #

**Review by prochigor — CHANGES_REQUESTED** [review]

File: cinema/models.py [hunk 3]
```diff
@@ -35,10 +39,17 @@ def full_name(self):
         return f"{self.first_name} {self.last_name}"
 
 
+def movie_image_file_path(instance, filename):
+    _, extention = os.path.splitext(filename)
+    filename = f"{slugify(instance.title)}-{uuid.uuid4()}.{extention}"
```
Comment (NEkropo1) [inline cinema/models.py]: you have excessive dot here

**Review by NEkropo1 — CHANGES_REQUESTED** [review]

File: cinema/models.py (hunk 3 above)
Comment (Daniil-Pankieiev) [reply cinema/models.py]: Fixed

**Review by Y-Havryliv — APPROVED** [review]

**Pull request closed without merging**